tests/integration/cli/scripts_test.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from os import path
+from shutil import rmtree
+from subprocess import check_call, check_output
+from tempfile import mkdtemp
+import time
+
+import git
+from mock import patch
+
+from dusty import commands
+from ...fixtures import basic_specs_fixture
+from ...testcases import DustyIntegrationTestCase
+
+class TestRestartCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestRestartCLI, self).setUp()
+        self.fake_repo_location = path.join(mkdtemp(), 'appa')
+        self._set_up_fake_local_repo(self.fake_repo_location)
+        basic_specs_fixture()
+        self.run_command('bundles activate bundle-a')
+        self.run_command('repos override github.com/app/a {}'.format(self.fake_repo_location))
+        self.run_command('up')
+
+    def tearDown(self):
+        self.run_command('bundles deactivate bundle-a')
+        self.run_command('repos manage github.com/app/a')
+        try:
+            self.run_command('stop --rm')
+        except Exception:
+            pass
+        rmtree(self.fake_repo_location)
+        super(TestRestartCLI, self).tearDown()
+
+    def test_basic(self):
+        self.exec_in_container('appa', 'touch /app/a/zoo')
+        self.run_command('scripts appa example')
+        self.assertEqual(self.exec_docker_process.stdout.read(), 'README.md  tee_dusty_always_fn_pipe_file  zoo\r\n')
```

Comment:
yeah, i'll just exec_in_container that it exists. That will save a lot of effort